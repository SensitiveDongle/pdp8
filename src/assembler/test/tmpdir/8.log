symbol var redefined; old value = 0x000, new value = 0x001
*** 1 errors in assembly
