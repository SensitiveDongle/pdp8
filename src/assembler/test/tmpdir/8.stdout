  0x000: 0x001    1: var, 1
  0x001: 0x002    2: var, 2
  0x002: 0x000    3: program, 0
  0x003: 0xF02    4: 	HLT
                  5: END program
